demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1135,7 +908,7 @@ export class DemoMeetingApp
           }
         } else {
           this.audioVideo.stopLocalVideoTile();
-          this.tileIndexToDemoVideoTile.get(DemoMeetingApp.LOCAL_VIDEO_TILE_INDEX).hide();
+        //   this.tileIndexToDemoVideoTile.get(DemoMeetingApp.LOCAL_VIDEO_TILE_INDEX).hide(); TODO
```

Comment:
what is the reason it has to be TODO :D

Reply:
Oops, this line of code isn't needed.  Switched to a comment.